lib/6to5/transformation/transformers/optional-self-contained.js

```diff
@@ -67,3 +69,10 @@ exports.ast = {
     });
   }
 };
+
+exports.MemberExpression = function (node, parent, scope, context, file) {
```

Comment:
A more elegant way to do this would be:

```javascript
exports.Identifier = function (node, parent) {
  if (node.name === "regeneratorRuntime" && t.isReferenced(node, parent)) {
    node.name = file._regeneratorId;
  }
};
```